### PR TITLE
Unify signup - READY FOR REVIEW #5

### PR DESCRIPTION
**Pull request by nitegeist**
## Overview

**What features/fixes does this PR include?**
This PR creates a page that unifies the signup of players, guilds and patrons
**Please provide the GitHub issue number**

Closes #

https://www.notion.so/meta-game/Improving-UX-02ea24e2fdc14cc4be0bebbede3b9487?p=e7842eac487f4e11b3495363133f09a2&pm=s

https://www.notion.so/meta-game/Rotating-1liners-4ab98ad7d7ab4821a8cfff7b0ec94c14

## Follow up Improvement Ideas

- [ ] please list any improvement/ideas

## Implementation

**Describe technical (nontrivial / non-obvious) parts of your code**

**Side effects**

## Assets

[Include screenshots/videos if it makes reviewing easier.]

### CODE REVIEW

**Review by Seroxdesign — CHANGES_REQUESTED** [review]

**Comment by Seroxdesign** [comment]
@nitegeist left you some comments in the discord channel, there's some visual issues that need to be tended to.

**Comment by alalonde** [comment]
Please reference an issue / artifact so I can test / review

**Comment by nitegeist** [comment]
> Please reference an issue / artifact so I can test / review

I'll have to make one. I don't think it exists

**Comment by Seroxdesign** [comment]
> Please reference an issue / artifact so I can test / review

https://www.notion.so/meta-game/Improving-UX-02ea24e2fdc14cc4be0bebbede3b9487?p=e7842eac487f4e11b3495363133f09a2&pm=s

**Review by alalonde — CHANGES_REQUESTED** [review]
Each tab should have its own URL so as not to break navigation and allow linking

**Comment by alalonde** [comment]
Is this really what we want it to look like? 
image

@davort @peth-yursick Can you take a look at this and suggest improvements? https://frontend-pr-1608-mjhnbmqqna-uk.a.run.app/onboarding

**Comment by nitegeist** [comment]
> Each tab should have its own URL so as not to break navigation and allow linking

This has been added with the latest commit

**Comment by alalonde** [comment]
> Each tab should have its own URL so as not to break navigation and allow linking

Close; when you land on /signup, it should redirect to /signup?tab=player

You can test by navigating back and forward, it should not skip any navigations

**Comment by nitegeist** [comment]
> when you land on /signup, it should redirect to /signup?tab=player

Done in the latest commit

**Comment by peth-yursick** [comment]
getting "service unavailable" on both this & test.metagame

**Review by Seroxdesign — CHANGES_REQUESTED** [review]

File: packages/web/components/Landing/Signup/index.tsx [hunk 1]
```diff
@@ -0,0 +1,894 @@
...
+      )}
+    </Button>
+  );
+});
+
+export const Signup: React.FC = () => {
+  const section = 'signup';
+  const router = useRouter();
+  const activeTab = useActiveTab();
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (nitegeist) [reply packages/web/components/Landing/Signup/index.tsx]: Removed the useEffect.

File: packages/web/lib/hooks/useActiveTab.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { useRouter } from 'next/router';
+
+export default function useActiveTab() {
```
[the remark this answers is not in the document]
Comment (nitegeist) [reply packages/web/lib/hooks/useActiveTab.ts:3]: This hook is used for quite a bit of things other than simply setting player as default active tab.

File: packages/web/components/Landing/Signup/index.tsx [hunk 3]
```diff
@@ -0,0 +1,894 @@
...
+                  </VStack>
+                </VStack>
+              </Box>
+            </TabPanel>
+          </TabPanels>
+        </Tabs>
+      </Container>
+      <Rain effectOpacity={0.2} />
+    </FullPageContainer>
+  );
```
[the remark this answers is not in the document]
Comment (nitegeist) [reply packages/web/components/Landing/Signup/index.tsx:693]: Done.

**Comment by Seroxdesign** [comment]
@nitegeist 

image

**Review by Seroxdesign — APPROVED** [review]

**Review by alalonde — APPROVED** [review]

**Pull request merged**